### PR TITLE
deps: Audit 2019-12-02

### PR DESCRIPTION
**Pull request by milesj**
to: @hayes @williaster 

Nothing significant in these changes, just transitive updates and some minor new ESLint rules.

### CODE REVIEW

File: packages/nimbus-common/constants.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ exports.EXTS_GROUP = '{ts,tsx,js,jsx}';
 
 exports.TJSX_EXT_PATTERN = /\.m?(t|j)sx?$/;
 
-exports.ASSET_EXT_PATTERN = /\.(ttf|eot|otf|svg|woff|woff2|mp3|png|jpg|jpeg|gif|ico|css|scss)$/;
+exports.ASSET_EXT_PATTERN = /\.(ttf|eot|otf|svg|woff|woff2|mp3|png|jpg|jpeg|gif|ico)$/;
```
Comment (milesj) [inline packages/nimbus-common/constants.js]: Allows consumers to add their own css loaders/plugins.

Comment (williaster) [reply packages/nimbus-common/constants.js]: nice, I think this will be good based on usage so far 👍

File: packages/nimbus/index.js [hunk 2]
```diff
@@ -58,6 +58,9 @@ module.exports = function cli(tool) {
       context.addArg(`./${srcFolder}`);
       context.addOption('--out-dir', context.args.esm ? './esm' : `./${buildFolder}`);
     }
+
+    // Changed in v7.7
+    driver.metadata.filterOptions = true;
```
Comment (williaster) [inline packages/nimbus/index.js]: so this filters any options to babel since unknown args like `--clean` or `--esm` now throw? 👍

Comment (milesj) [reply packages/nimbus/index.js]: It will filters all options that Babel doesn't support natively.

**Pull request merged**